django_extensions/management/commands/runserver_plus.py

```diff
@@ -149,9 +156,44 @@ def execute(self, sql, params=()):
         except ImportError:
             raise CommandError("Werkzeug is required to use runserver_plus.  Please visit http://werkzeug.pocoo.org/ or install via pip. (pip install Werkzeug)")
 
+        try:
+            from django_pdb.middleware import PdbMiddleware
+
+            pdb_option = options.get('pdb', False)
+            ipdb_option = options.get('ipdb', False)
+            pm = options.get('pm', False)
+
+            # Add pdb middleware if --pdb is specified or if in DEBUG mode
+            middleware = 'django_pdb.middleware.PdbMiddleware'
+            if ((pdb_option or settings.DEBUG) and
```

Comment:
Yeah, I couldn't figure out what purpose that served in pdb-django but I forgot to try removing it.